modules/ossm-deploy-single-tenant.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+[id="ossm-deploy-single-tenant_{context}"]
+= Single tenancy model
+
+[role="_abstract"]
+In Istio, a tenant is a group of users that share common access and privileges for a set of deployed workloads. Tenants can be used to provide a level of isolation between different teams.
```

Comment:
```suggestion
In Istio, a single control plane is used to manage services and workloads across an entire Kubernetes cluster.  Users must apply custom approaches to segregate access to services and workloads from other services and workloads within the cluster.  This can be done using `NetworkPolicy` resources, `AuthorizationPolicy` resources, `exportTo` annotations on `Service` resources, and/or `exportTo` fields on various istio.io resources.
```

Reply:
Not sure what to do with this suggestion, since we don't really support single tenant in the product.

Reply:
Yeah, and now I don't remember what I was thinking wrt this.  Maybe we just delete this whole single tenant thing, as it's not supported at all by ossm.

Reply:
The suggestion here makes it sound to me like there's a central control plane that is responsible for managing services and workloads across the entire Kubernetes cluster. What about:

```suggestion
In Istio, a tenant is a group of users that share common access and privileges for a set of deployed workloads. Tenants can be used to provide a level of isolation between different teams. You can segregate access to different tenants using `NetworkPolicies`, `AuthorizationPolicies`, and `exportTo` annotations on istio.io or service resources. 
```

Reply:
I'm going to go with Brian's suggestion for now.  I feel like if we're going to talk about multi-tenant, that we need at least a tiny blurb about single-tenant, even if we don't support it.